R/pkg/R/DataFrame.R

```diff
@@ -2732,6 +2732,20 @@ setMethod("union",
             dataFrame(unioned)
           })
 
+#' Return a new SparkDataFrame containing the union of rows
+#'
+#' This is an alias for `union`.
```

Comment:
If the goal is for this to be like other *All, this should go into a separate doc page, plus seealso, example etc.

The way this was written, as it was a deprecated function, this doc page merge with union - as it is, none of the text above will show up and also unionAll will not be listed in method index list.

Reply:
also backtick doesn't format with roxygen2. this should be 
```
This is an alias for \code{union}.
```

Reply:
I see. Instead of directly copying the comments back, we should follow intersectAll. Opened a ticket: https://issues.apache.org/jira/browse/SPARK-26189